mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/AttachmentLoader.java

```diff
@@ -43,12 +43,12 @@ public AttachmentLoader(AttachmentMapper attachmentMapper) {
 
     public Collection<MessageAttachment> getAttachments(Set<CassandraMessageDAO.MessageAttachmentRepresentation> attachmentRepresentations) {
 
-        Map<AttachmentId, CassandraMessageDAO.MessageAttachmentRepresentation> attachmentIds = attachmentRepresentations.stream()
+        Map<AttachmentId, CassandraMessageDAO.MessageAttachmentRepresentation> attachmentRepresentationByIds = attachmentRepresentations.stream()
                 .collect(Guavate.toImmutableMap(CassandraMessageDAO.MessageAttachmentRepresentation::getAttachmentId, Function.identity()));
 
-        Map<AttachmentId, Attachment> attachmentByIdMap = attachmentsById(attachmentIds.keySet());
+        Map<AttachmentId, Attachment> attachmentById = attachmentsById(attachmentRepresentationByIds.keySet());
 
-        return MapMerger.merge(attachmentByIdMap, attachmentIds, this::constructMessageAttachment).values();
+        return MapMerger.merge(attachmentById, attachmentRepresentationByIds, this::constructMessageAttachment).values();
```

Comment:
sorry to be picky about that, but your are not consistent with the usage of final 's'
after that you can rebase and autosquash for me